visualization/react-lib/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "quantum-visualization",
+  "version": "1.0.0",
+  "description": "This is a visualization library to display Azure Quantum job result.",
+  "main": "dist/main.js",
+  "types": "src/index.ts",
+  "scripts": {
+    "build": "webpack --mode development",
+    "build:prod": "webpack --mode production",
+    "tests": "jest --config ./test-config/jest.config.js --verbose=true --coverageReporters=cobertura",
+    "testsonly": "jest --config ./test-config/jest.config.js --verbose=true --no-coverage",
+    "updatetests": "jest --config ./test-config/jest.config.js --verbose=true --no-coverage --updateSnapshot",
+    "lint": "eslint .  --ext .js,.ts,.tsx",
+    "lint-fix": "eslint --fix .  --ext .js,.ts,.tsx"
+  },
+  "jest": {
+    "transformIgnorePatterns": "[`/node_modules/('@table-library/react-table-library/theme', '@table-library/react-table-library/material-ui', '@table-library/react-table-library/table', '@mui/material', '@mui/icons-material/Info')].join('|'))`]",
+    "moduleNameMapper": {
+      "d3": "<rootDir>/node_modules/d3/dist/d3.min.js",
+      "\\.(css|less)$": "<rootDir>/__mocks__/styleMock.js"
+    }
+  },
+  "license": "MIT",
+  "dependencies": {
+    "@types/d3": "^7.4.0",
```

Comment:
I beleive that @types should only contain typescript description, which should be devDependencies

Reply:
that makes sense, will update